### PR TITLE
Tests for measuring with `time`

### PR DESCRIPTION
**Pull request by MasloMaslane**

### CODE REVIEW

**Comment by MasloMaslane** [comment]
I don't think that any new tests are needed for measuring with `time`.

**Review by tonowak — APPROVED** [review]
Nice! That's a great way of testing it without writing duplicate code, I like it!

Don't merge this PR until the other branch will be approved. Otherwise the files from this PR will need to be reviewed/approved also in the other PR.

**Pull request merged**
